Add documentation for new Rabbit Air integration

al-s opened the pull request:
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
Add documentation for the new RabbitAir integration.


## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/66130
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3150
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

al-s committed:
Add documentation for new Rabbit Air integration

homeassistant commented:
Hi @al-s, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#21560).

Once you do that we will be able to review and accept this pull request.

Thanks!

al-s committed:
Remove options flow

Scan interval should not be part of integration configuration. This was
the only option, so the options flow can be fully removed.

frenck commented on source/_integrations/rabbitair.markdown:
```diff
@@ -0,0 +1,58 @@
+ha_category:
+  - Fan
+ha_iot_class: Local Polling
+ha_release: 2022.3
```
```suggestion
ha_release: 2024.2
```

frenck commented on source/_integrations/rabbitair.markdown:
```diff
@@ -0,0 +1,58 @@
+  - fan
+---
+
+The Rabbit Air integration allows you to control your air purifier over the local network. The following device models are currently supported:
+
+- MinusA2 (2-nd generation)
+- A3
+
+The fan platform of this integration allows you to turn on/off the unit, select the preset mode or set the speed manually.
```
```suggestion
The Rabbit Air integration lets you control your air purifier over the local network. The following device models are currently supported:

- MinusA2 (2-nd generation)
- A3

The fan platform of this integration allows you to turn the unit on/off, select the preset mode, or set the speed manually.
```

frenck commented on source/_integrations/rabbitair.markdown:
```diff
@@ -0,0 +1,58 @@
+
+The fan platform of this integration allows you to turn on/off the unit, select the preset mode or set the speed manually.
+
+## Retrieving the Access Token
```
```suggestion
## Pre-requisites
```

frenck commented on source/_integrations/rabbitair.markdown:
```diff
@@ -0,0 +1,58 @@
+
+To set up the integration, you will need to know a device address and an access token.
+
+1. Open the Rabbit Air mobile app. You will see a list of devices connected to your account.
+2. Tap the list item and the device control page will open.
+3. On the device page tap the `Edit` button. You will see a page with the device location and name settings.
+4. On this page, quickly tap on "Serial Number" several times until you see two more lines that were previously hidden. The first is the device ID, and the second is the access token.
```
```suggestion
1. Open the Rabbit Air mobile app. You will see a list of devices connected to your account.
2. Tap the list item, and the device control page will open.
3. On the device page, select the **Edit** button. You will see a page with the device location and name settings.
4. On this page, quickly tap **Serial Number** several times until you see two more previously hidden lines. The first is the device ID, and the second is the access token.
```

frenck commented on source/_integrations/rabbitair.markdown:
```diff
@@ -0,0 +1,58 @@
+  </a>
+</p>
+
+Then you can use `abcdef1234_123456789012345678.local` as the "Host" and `0123456789ABCDEF0123456789ABCDEF` as the "Access Token".
```
```suggestion
Then you can use `abcdef1234_123456789012345678.local` as the **Host** and `0123456789ABCDEF0123456789ABCDEF` as the **Access Token**.
```

al-s committed:
Apply suggestions from code review

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

c0ffeeca7 commented on source/_integrations/rabbitair.markdown:
```diff
@@ -0,0 +1,58 @@
+
+The fan platform of this integration allows you to turn the unit on/off, select the preset mode, or set the speed manually.
+
+## Pre-requisites
```
```suggestion
## Prerequisites
```

c0ffeeca7 committed:
tiny tweak

c0ffeeca7 reviewed (approved):
Thank you, @al-s :+1:
